src/app/cube/shared/learning-object/learning-object.component.ts

```diff
@@ -118,7 +118,7 @@ export class LearningObjectListingComponent implements OnInit, OnChanges, OnDest
     // The top regex is used for matching tags such as <br />
     // The bottom regex will catch tags such as </p>
     str = str.replace(/<[0-z\s\'\"=]*[\/]+>/gi, ' ');
-    return str.replace(/<[\/]*[0-z\s\'\"=]+>/gi, ' ');
+    return str.replace(/<(?!\/?a\s+href="[^"\/\"]*")[^>]+>/gi, ' ');
```

Comment:
Let's hold off on this merge this until we talk about if there is more to this. I think this covers one case, but there is a few others that we noted in the Shortcut story.